### ISSUE TITLE
Does this client support syncing sessions across all available memcached nodes?

### ISSUE THREAD

**Issue by Cull-Methi** [magro/memcached-session-manager#343]
I'd like to use this client to make the memcached installation have session redundancy and be HA - does this client support this?

I see in the readme.md: 

> Also memcached failover (memcached crash) is supported via migration of sessions.

But I can't seem to find it in the SetupAndConfiguration page. Is it just supported automatically?

**Comment by magro** [comment]
> Does this client support syncing sessions across all available memcached nodes?

No, in sticky mode msm stores a session only in a single memcached node (as "backup", to support failover). In non-sticky mode msm stores a session in a primary memcached node and for backup in an additional node. 

> I'd like to use this client to make the memcached installation have session redundancy and be HA - does this client support this?

I'd say yes :-) 

> But I can't seem to find it in the SetupAndConfiguration page.

What are you missing?

**Comment by Cull-Methi** [comment]
Ok, I think I understand now. In order to get a session stored in multiple memcached nodes, I should set sticky=false when I add it to the context.xml for tomcat, yes?

**Comment by magro** [comment]
You should only set sticky=false if you are not going to use sticky sessions. I.e. you must configure your loadbalancer appropriately, this also depends on the web framework you're using.
